lab_1/main.py

```diff
@@ -107,7 +248,14 @@ def load_profile(path_to_file: str) -> dict or None:
     :param path_to_file: a path
     :return: a dictionary with three keys – name, freq, n_words
     """
-    pass
+    if not isinstance(path_to_file, str):
+        return None
+    try:
```

Comment:
why try/except? what are they for?

Reply:
If the path will not be found, it helps to avoid an error and to return None

Reply:
but why to use it? I don't get it
A simple `If` will do